[2022-04-04 16:28:44,205] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: s3_uni_flores.upload_to_s3 manual__2022-04-04T21:28:36.104534+00:00 [queued]>
[2022-04-04 16:28:44,213] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: s3_uni_flores.upload_to_s3 manual__2022-04-04T21:28:36.104534+00:00 [queued]>
[2022-04-04 16:28:44,213] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-04-04 16:28:44,213] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-04-04 16:28:44,213] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-04-04 16:28:44,458] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): upload_to_s3> on 2022-04-04 21:28:36.104534+00:00
[2022-04-04 16:28:44,460] {standard_task_runner.py:52} INFO - Started process 6828 to run task
[2022-04-04 16:28:44,464] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 's3_uni_flores', 'upload_to_s3', 'manual__2022-04-04T21:28:36.104534+00:00', '--job-id', '359', '--raw', '--subdir', 'DAGS_FOLDER/dag_s3_universidad_de_las_flores.py', '--cfg-path', '/tmp/tmpa0fkyje0', '--error-file', '/tmp/tmphtnid6l9']
[2022-04-04 16:28:44,464] {standard_task_runner.py:77} INFO - Job 359: Subtask upload_to_s3
[2022-04-04 16:28:44,811] {logging_mixin.py:109} INFO - Running <TaskInstance: s3_uni_flores.upload_to_s3 manual__2022-04-04T21:28:36.104534+00:00 [running]> on host LAPTOP-5HJG55H7.localdomain
[2022-04-04 16:28:45,148] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=s3_uni_flores
AIRFLOW_CTX_TASK_ID=upload_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2022-04-04T21:28:36.104534+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-04T21:28:36.104534+00:00
[2022-04-04 16:28:45,157] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-04-04 16:28:46,489] {logging_mixin.py:109} INFO - /home/lowenhard/airflow/files/ET_Univ_DE_Flores.txt ET_Univ_DE_Flores.txt cohorte-marzo-77238c6a
[2022-04-04 16:28:46,490] {logging_mixin.py:109} INFO - Created Connection
[2022-04-04 16:28:47,725] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2022-04-04 16:28:47,735] {logging_mixin.py:109} INFO - Session(region_name=None)
[2022-04-04 16:28:47,735] {logging_mixin.py:109} INFO - True
[2022-04-04 16:28:47,735] {dag_s3_universidad_de_las_flores.py:30} INFO - Checking s3 connection
[2022-04-04 16:28:47,738] {python.py:175} INFO - Done. Returned value was: None
[2022-04-04 16:28:47,744] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=s3_uni_flores, task_id=upload_to_s3, execution_date=20220404T212836, start_date=20220404T212844, end_date=20220404T212847
[2022-04-04 16:28:47,909] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-04 16:28:47,925] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
